common/tier4_autoware_utils/include/tier4_autoware_utils/ros/polling_reciever.hpp

```diff
@@ -0,0 +1,55 @@
+// Copyright 2024 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef TIER4_AUTOWARE_UTILS__ROS__POLLING_RECEIVER_HPP_
+#define TIER4_AUTOWARE_UTILS__ROS__POLLING_RECEIVER_HPP_
+
+#include <rclcpp/rclcpp.hpp>
+
+namespace tier4_autoware_utils
+{
+
+template <typename T>
+class InterProcessPollingReceiver
+{
+private:
+  typename rclcpp::Subscription<T>::SharedPtr subscriber;
+
+public:
+  explicit InterProcessPollingReceiver(rclcpp::Node * node, const std::string & topic_name)
```

Comment:
explict is not required for the two arguments.

Reply:
Yes. we can remove `explicit`. 
But explicit makes sense even for multi arguments constructor.

Reply:
determined to maintain explicit by a book written by Bjarne Stroustrup